test/libsolidity/smtCheckerTests/overflow/overflow_sum.sol

```diff
@@ -14,6 +14,5 @@ contract C
 	}
 }
 // ----
-// Warning 2529: (109-116): Overflow (resulting value larger than 255) happens here
```

Comment:
@ekpyron I hadn't noticed but this warning was added by the first commit in this PR, but it shouldn't be reported because of the always reverts thing, so it's good it's removed now.